src/bun.js/webcore/streams.zig

```diff
@@ -96,10 +97,21 @@ pub const ReadableStream = struct {
         return this.value;
     }
 
+    pub fn reloadTag(this: *ReadableStream, globalThis: *JSC.JSGlobalObject) void {
+        if (ReadableStream.fromJS(this.value, globalThis)) |stream| {
+            this.* = stream;
+        } else {
+            this.value.unprotect();
+            this.* = .{ .ptr = .{ .Invalid = {} }, .value = .zero };
+        }
+    }
+
     pub fn toAnyBlob(
         stream: *ReadableStream,
         globalThis: *JSC.JSGlobalObject,
     ) ?JSC.WebCore.AnyBlob {
+        stream.reloadTag(globalThis);
```

Comment:
This is the main cause 

We internally store the `ReadableStream` on the `PendingValue`, and the pointer becomes stale. It is a use-after-free where the `ptr` has been detached, but the existing `PendingValue` hasn't been updated to reflect that. To address this, we reload the tag.